extensions/typescript-language-features/src/typeScriptServiceClientHost.ts

```diff
@@ -150,6 +151,60 @@ export default class TypeScriptServiceClientHost extends Disposable {
 					standardFileExtensions: [],
 				}, onCompletionAccepted);
 			}
+			// TODO: Still probably isn't the right place for this
+			const ext = vscode.extensions.getExtension('github.copilot');
+			if (!ext) {
+				vscode.window.showErrorMessage(vscode.l10n.t('Could not find related-files extension'));
+				return;
+			}
+			await ext.activate();
+			const relatedAPI = ext.exports as {
+				registerRelatedFilesProvider(
+					providerId: { extensionId: string; languageId: string },
+					callback: (uri: vscode.Uri) => Promise<{ entries: vscode.Uri[]; traits?: { name: string; value: string }[] }>
+				): void;
+			} | undefined;
+			if (relatedAPI) {
+				for (const languageId of jsTsLanguageModes) {
+					const id = {
+						extensionId: 'vscode.typescript-language-features',
+						languageId
+					};
+					relatedAPI.registerRelatedFilesProvider(id, async uri => {
```

Comment:
Does copilot pass a cancellation token? If not, we should request that it does

Reply:
I'll go add that, but I'd like to not block this PR on it, since it will be functional without cancellation (and I couldn't easily figure out the cancellation story over there, so it may not be trivial).